doc/reference/javascript_reference.rst

```diff
@@ -943,10 +937,10 @@ aware of its existence.
 
 There are a few registries available in the system:
 
-- field registry (exported by 'web.field_registry'). The field registry contains
-    all field widgets known to the web client.  Whenever a view (typically, form,
-    or list/kanban) needs a field widget, this is where it will look. A typical
-    use case look like this:
+- field registry (exported by 'web.field_registry')
+    The field registry contains all field widgets known to the web client.
+    Whenever a view (typically form or list/kanban) needs a field widget, this
+    is where it will look. A typical use case look like this:
```

Comment:
Using a bullet-point list seems unnecessary if we're using definition lists as well, remove the bullet points?

Also `'web.field_registry'` should probably be in double-backticks instead of quotes, or maybe 

    :js:data:`web.field_registry`

if the rendering isn't awful.

Reply:
The rendering looks better so I've used it. I've also replaced a single quoted `require` that's used inline in a paragraph.
I agree on the redundant bullet-point + definition list pattern, I've removed it throughout the document. That does blow up the diff a bit, merging is going to be fun :-P